RELEASE.md

```diff
@@ -0,0 +1,81 @@
+Release Instructions
+====================
+
+Set up Sonatype Account
+-----------------------
+* Sign up for a Sonatype JIRA account [here](https://issues.sonatype.org)
+* Click *Sign Up* in the login box, follow instructions
+
+Get access to repository
+------------------------
+* Go to [community support](https://issues.sonatype.org/browse/OSSRH)
+* Ask for publish rights by creating an issue similar to [this one](https://issues.sonatype.org/browse/OSSRH-16798)
+
+Set up PGP keys
+---------------
+* Generate the key ```gpg --gen-key```
+
+  * Keep the defaults, but specify a passphrase
+  * The passphrase can be random; you just need to remember it long enough to finish the next step
+  * One way to make a random passphrase: ```base64 /dev/urandom | head -c20 | (cat;echo)```
+
+* Find the ID of your public key ```gpg --list-secret-keys```
+  * Look for the line with format ```sec   2048R/ABCDEFGH 2015-11-17```
+  * The ```ABCDEFGH``` is the ID for your public key
+
+* Upload your public key to a public server: ```gpg --send-keys --keyserver hkp://pgp.mit.edu ABCDEFGH```
+
+Create a Maven settings file
+----------------------------
+* Create a file at ```$HOME/.m2/settings.xml``` with your passphrase and your sonatype username and password
+```
+<settings>
+  <profiles>
+    <profile>
+      <id>ossrh</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <properties>
+        <gpg.executable>gpg</gpg.executable>
+        <gpg.passphrase>[the password for your gpg key]</gpg.passphrase>
+      </properties>
+    </profile>
+  </profiles>  
+  <servers>
+    <server>
+      <id>ossrh</id>
+      <username>[your sonatype account name]</username>
+      <password>[your sonatype account password]</password>
+    </server>
+  </servers>
+</settings>
+```
+
+Deploy to Sonatype
+------------------
+* Update all ```pom.xml``` files in the package to the release version you want. Sumit a pull request, get it reviewed, and submit
+* ```mvn clean install deploy -DperformRelease=true```
+* Verify the publishment [here](https://oss.sonatype.org/#nexus-search;quick~com.google.auth)
+  * If there is a problem, undo by ```mvn nexus-staging:drop```
+* ```mvn nexus-staging:release -DperformRelease=true```
+* Update all ```pom.xml``` files to the new snapshot version
+
+Publish the release
+-------------------
+* Go to [Sonatype](https://oss.sonatype.org/) and log in
```

Comment:
If I recall, the above Deploy to Sonatype should also publish it, so this could incorrectly imply you need to also the manual step. I think the simplest thing might be to not have this set of separate manual steps under "Publish the Release". However, we should also check with @garrettjonesgoogle since he added this section to the source document.